assembly.py

```diff
@@ -241,7 +241,9 @@ def trim_rmdup_subsamp_reads(inBam, clipDb, outBam, n_reads=100000):
                 os.unlink(f[i])
 
     # multiply counts so all reflect individual reads
-    return (n_input * 2, n_trim * 2, n_rmdup * 2, n_output, n_paired_subsamp * 2, n_unpaired_subsamp)
+    outStats_tuple = (n_input * 2, n_trim * 2, n_rmdup * 2, n_output, n_paired_subsamp * 2, n_unpaired_subsamp)
+    if outStatsFile: util.file.dump_file(outStatsFile, outStats_tuple)
```

Comment:
Maybe this should be a csv/tsv file that includes a header to disambiguate the column values. Something like this?
```python
with open(outStatsFile, 'w') as f:
    csvwriter = csv.writer(f, delimiter='\t', quoting=csv.QUOTE_MINIMAL)
    csvwriter.writerow(["paired_input","paired_after_trimmomatic","paired_after_prinseq_rmdup","individual_output_reads", "paired_output_reads", "unpaired_output_reads"])
    csvwriter.writerow(outStats_tuple)
```